radix-transactions/src/builder/transaction_builder.rs

```diff
@@ -54,15 +54,19 @@ impl TransactionV1Builder {
 
     pub fn sign<S: Signer>(mut self, signer: &S) -> Self {
         let intent = self.transaction_intent();
-        let prepared = intent.prepare().expect("Intent could be prepared");
+        let prepared = intent
+            .prepare(PreparationSettings::latest_ref())
+            .expect("Intent could be prepared");
         self.intent_signatures
             .push(signer.sign_with_public_key(&prepared.transaction_intent_hash()));
         self
     }
 
     pub fn multi_sign<S: Signer>(mut self, signers: &[&S]) -> Self {
         let intent = self.transaction_intent();
-        let prepared = intent.prepare().expect("Intent could be prepared");
+        let prepared = intent
+            .prepare(PreparationSettings::latest_ref())
+            .expect("Intent could be prepared");
```

Comment:
"could **not**"  :)

Reply:
It's actually correct-ish. https://doc.rust-lang.org/std/option/enum.Option.html#recommended-message-style

The recommendations say you should state what your expectation was that wasn't met. Ideally using the word "should". So according to the rust recommendations I think we should use "Intent should be preparable". So will change to that :+1:

Reply:
I didn't know about that recommendation! Sounds good :+1: